bundles/org.corpus_tools.hexatomic.core/src/main/java/org/corpus_tools/hexatomic/core/ui/AboutDialog.java

```diff
@@ -0,0 +1,176 @@
+/*-
+ * #%L
+ * org.corpus_tools.hexatomic.core
+ * %%
+ * Copyright (C) 2018 - 2020 Stephan Druskat, Thomas Krause
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+package org.corpus_tools.hexatomic.core.ui;
+
+import org.eclipse.jface.dialogs.Dialog;
+import org.eclipse.jface.dialogs.IDialogConstants;
+import org.eclipse.jface.resource.FontDescriptor;
+import org.eclipse.jface.resource.JFaceResources;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.SelectionAdapter;
+import org.eclipse.swt.events.SelectionEvent;
+import org.eclipse.swt.graphics.Font;
+import org.eclipse.swt.graphics.Point;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.program.Program;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Control;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.swt.widgets.Link;
+import org.eclipse.swt.widgets.Shell;
+import org.osgi.framework.FrameworkUtil;
+import org.osgi.framework.Version;
+
+public class AboutDialog extends Dialog {
+
+  private final Font headerFont;
+  private final Font versionFont;
+
+
+  /**
+   * Create the dialog.
+   * 
+   * @param parentShell The parent
+   */
+  public AboutDialog(Shell parentShell) {
+    super(parentShell);
+
+    Font dialogFont = JFaceResources.getDialogFont();
+    FontDescriptor headerFontDescriptor =
+        FontDescriptor.createFrom(dialogFont).setHeight(32);
+    headerFont = headerFontDescriptor.createFont(parentShell.getDisplay());
+
+    FontDescriptor boldFontDescriptor =
+        FontDescriptor.createFrom(dialogFont).setStyle(SWT.BOLD).setHeight(16);
+    versionFont = boldFontDescriptor.createFont(parentShell.getDisplay());
+
+  }
+
+  @Override
+  protected void configureShell(Shell newShell) {
+    super.configureShell(newShell);
+    newShell.setText("About Hexatomic");
+  }
+
+  private static String getFullVersion() {
+    Version v = FrameworkUtil.getBundle(AboutDialog.class).getVersion();
+    return String.format("%d.%d.%d", v.getMajor(), v.getMinor(), v.getMicro());
+  }
+
+  private static String getShortVersion() {
+    Version v = FrameworkUtil.getBundle(AboutDialog.class).getVersion();
+    return String.format("%d.%d", v.getMajor(), v.getMinor());
+  }
+
```

Comment:
```suggestion


private static String getQualifiedVersion() {
  Version v = FrameworkUtil.getBundle(AboutDialog.class).getVersion();
  return String.format("%d.%d.%d-%s", v.getMajor(), v.getMinor(), v.getMicro(), v.getQualifier());
}


```

Add this to get access to a qualified version string.